### PR TITLE
fix(starr): Netflix CF to match NetflixHD and NetflixUHD in title

### PR DESCRIPTION
**Pull request by lucasreiners**
# Pull Request

## Purpose

German releases often use NetflixHD in the release title, which is currently not matched by the custom format.
Basically the same as in https://github.com/TRaSH-Guides/Guides/pull/1829

## Approach

Added optional HD suffix to the custom format.

Regex Test: https://regex101.com/r/QpfWrs/1

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by TRaSH-** [comment]
Don't they also use netflixuhd ?

**Comment by lucasreiners** [comment]
Actually never saw that, so I am not sure, but if you would like, I could optionally add that, too

**Comment by TRaSH-** [comment]
> Actually never saw that, so I am not sure, but if you would like, I could optionally add that, too

yeah just add it, i think i found part of it in the Sonarr source code

**Comment by lucasreiners** [comment]
Updated regex test:
https://regex101.com/r/QpfWrs/2

**Comment by TRaSH-** [comment]
@nuxencs could you review this one ?

File: docs/json/radarr/cf/nf.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "\\b(nf|netflix)\\b"
+        "value": "\\b(nf|netflix(u)?(hd)?)\\b"
```
Comment (nuxencs) [inline docs/json/radarr/cf/nf.json:12]: https://regex101.com/r/pWZokU/1
```suggestion
        "value": "\\b(nf|netflix(u?hd)?)\\b"
```

File: docs/json/sonarr/cf/nf.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "\\b(nf|netflix)\\b"
+        "value": "\\b(nf|netflix(u)?(hd)?)\\b"
```
Comment (nuxencs) [inline docs/json/sonarr/cf/nf.json:16]: https://regex101.com/r/pWZokU/1
```suggestion
        "value": "\\b(nf|netflix(u?hd)?)\\b"
```

**Review by nuxencs — CHANGES_REQUESTED** [review]
just a small simplification

**Comment by lucasreiners** [comment]
Thank you, simplicfication still works as expected:
https://regex101.com/r/QpfWrs/3

**Review by nuxencs — APPROVED** [review]

**Pull request merged**